install/kubernetes/cilium/values.yaml.tmpl

```
@@ -2200,12 +2200,16 @@ envoy:
   # Ref: https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy
   dnsPolicy: ~
 
+  # -- Configure Cilum Envoy Prometheus options.
```

Nit:
```suggestion
  # -- Configure Cilium Envoy Prometheus options.
```
(Re-gen files).